Libplanet.Action.Tests/Common/DumbAction.cs

```diff
@@ -81,6 +72,28 @@ public IValue PlainValue
             }
         }
 
+        public static DumbAction Create(
+            (Address At, string Item)? append = null,
+            (Address? From, Address? To, BigInteger Amount)? transfer = null,
+            IEnumerable<Validator>? validators = null,
+            bool recordRandom = false)
+        {
+            if (transfer is { } t && t.From is null && t.To is null)
```

Comment:
The signature allows it. The type for transfer is now `(Address?, Address?, BigInteger)?`.

Reply:
This is because `Transfer` will be used for mint and burn as well. The reason I didn't separate mint or burn as separate properties is
- Once you have transfer, mint, and/or burn allowed all in the same action, the order of operation is no longer well-defined.
- If they are made exclusive, you have to check there is less than or equal to one non-`null` operation. This makes the design more complicated. `DumbAction`'s specification should be as simple as possible.

Reply:
I'm understood. You've already mentioned at the summary. 🙏